timescaledb/how-to-guides/ingest-data.md

```diff
@@ -1,100 +1,79 @@
 # Ingesting data
-TimescaleDB supports standard SQL inserts. For more information about how to use
-SQL to write data into TimescaleDB, see the [Writing Data][writing-data]
-section.
-
-You can choose to leverage existing third party tools to build data ingest
-pipelines that increase ingest rates by performing batch writes, instead
-inserting data one row or metric at a time. Any tool that can read or write to
-PostgreSQL also works with TimescaleDB.
+By defult, TimescaleDB supports standard SQL inserts. Additionally, you can use
+third party tools to build data ingest pipelines. A data ingest pipeline can
+increase your data ingest rates by using batch writes, instead of inserting data
+one row or metric at a time. Any tool that can read or write to PostgreSQL also
+works with TimescaleDB.
 
 This section covers some popular frameworks and systems used in conjunction with
 TimescaleDB.
 
+For more information about how to use standard SQL insert queries to write data
+into TimescaleDB, see the [Writing Data][writing-data] section.
+
 ## Prometheus
-Prometheus is a popular tool used to monitor infrastructure metrics. It can
-scrape any endpoints that expose metrics in a Prometheus-compatible format. The
-metrics are stored in Prometheus and can be queried using PromQL. Prometheus
-itself is not built for long-term metrics storage, and instead, supports a
-variety of remote storage solutions.
-
-We developed a [Promscale][promscale-blog] that allows Prometheus to use
-TimescaleDB as a remote store for long-term metrics. Promscale supports
-both PromQL and SQL, PromQL queries can be directed to the Promscale
-endpoint or Prometheus instance and the [SQL API][promscale-sql] can
-be accessed by connecting to Timescale directly. It also offers other
-native time-series capabilities, such as automatically
+Prometheus is used to monitor infrastructure metrics. It scrapes any endpoints
+that can expose metrics in a compatible format. The metrics are stored in
+Prometheus, and you can query them using the PromQL query language. Prometheus
+is not intended to be used for long-term metrics storage, but it supports a
+variety of remote storage solutions that can be used for that purpose.
+
+TimescaleDB can use Prometheus as a remote store for long-term metrics, by using
+[Promscale][promscale]. Promscale supports both PromQL and SQL queries. PromQL
```

Comment:
Don't know if this is just a problem with the dev environment, but this gave me a 403 error. All the other links work.

Reply:
@jacobprall ??

Reply:
@charislam change to /promscale/:currentVersion:/ for the URL